src/timeline/TimelineViewManager.h

```diff
@@ -98,6 +97,7 @@ class TimelineViewManager : public QObject
     void openInviteUsersDialog(InviteesModel *invitees);
     void openProfile(UserProfile *profile);
     void showImagePackSettings(ImagePackListModel *packlist);
+    void openLeaveRoomDialog(QString roomid);
```

Comment:
I would prefer to have this in the Nheko object too (same on the other PRs, I suppose). Could you then also rebase and resolve the conflicts in the res.qrc?

Reply:
Sure, of course!